src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+public class DukeException extends Exception {
+
+    private String dukeError;
+
+    /**
+     * Constructor for a DukeException

```

Comment:
There should be a space between headers and parameter sections in the Javadoc comment